tests/Http/SignupTest.php

```diff
@@ -382,6 +382,9 @@ public function testSignupIndexWithIncludedUserAsAdmin()
         $post = factory(Post::class)->create();
         $signup = $post->signup;
 
+        // Mock the Northstar API calls.
+        $this->northstarMock;
```

Comment:
These lines aren't doing anything & can be removed, [like in `PostTest`](
https://github.com/DoSomething/rogue/pull/749/files/76985add32c27f1f95beb0a1e1ed7fb9645488bd#diff-ec1f9ba9f2cdaf675bcdfdd5136f75f).

Reply:
ah shoot sorry about that - my bad! 🤦‍♀️ Updated now!